### DISCUSSION TITLE
Implicit or explicit FEM implementation?

### DISCUSSION THREAD

**Discussion by FluffoX** [NVIDIA-Omniverse/PhysX#186] [Q&A] [answered]
Hello everyone, 
i am currently working intensively with the deformable body component of the physx engine and have encountered some limitations. It seems that these limitations (e.g. problems with very stiff bodies) are due to the way the FEM is implemented. In the literature i found concrete statements that these problems usually occur with the explicit FEM. Unfortunately i could not find any reliable source or statement about what kind of FEM (implicit, explicit or something else) is implemented in the physix engine. Can anybody help me with this?
Thanks for reading!

**Comment by jumyungc** [comment, marked as answer]
Hello. PhysX softbody is based on a (close) approximation of implicit Euler as claimed in "XPBD: Position-Based Simulation of Compliant Constrained Dynamics" with additional improvements.

**Reply by FluffoX** [reply]
Thanks for the quick reply, I had just started looking into this.  It's good to have that confirmed!
